Regression in master because of change in pandas

The master branch isn't passing Travis tests, so PR branches are also failing. This shows up e.g. in this CRON build: https://travis-ci.org/github/cgre-aachen/gempy/builds/714780814 , where the error was

```
____________ ERROR collecting test/test_core/test_data_mutation.py _____________
test/test_core/test_data_mutation.py:11: in <module>
    mm.add_surfaces(['surface1', 'foo1', 'foo2', 'foo3'])
gempy/core/model.py:737: in add_surfaces
    self._surfaces.add_surface(surface_list, update_df)
gempy/core/data.py:596: in add_surface
    self.map_series()
gempy/core/data.py:760: in map_series
    self.sort_surfaces()
gempy/core/data.py:678: in sort_surfaces
    self.df.sort_values(by=['series', 'order_surfaces'], inplace=True)
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/frame.py:5301: in sort_values
    indexer, axis=self._get_block_manager_axis(axis), verify=False
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/internals/managers.py:1415: in take
    new_axis=new_labels, indexer=indexer, axis=axis, allow_dups=True
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/internals/managers.py:1259: in reindex_indexer
    for blk in self.blocks
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/internals/managers.py:1259: in <listcomp>
    for blk in self.blocks
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/internals/blocks.py:1720: in take_nd
    new_values = self.values.take(indexer, fill_value=fill_value, allow_fill=True)
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/core/series.py:829: in take
    nv.validate_take(tuple(), kwargs)
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/compat/numpy/function.py:68: in __call__
    validate_kwargs(fname, kwargs, self.defaults)
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/util/_validators.py:148: in validate_kwargs
    _check_for_invalid_keys(fname, kwargs, compat_args)
../../../miniconda/envs/tenv/lib/python3.7/site-packages/pandas/util/_validators.py:122: in _check_for_invalid_keys
    raise TypeError(f"{fname}() got an unexpected keyword argument '{bad_arg}'")
E   TypeError: take() got an unexpected keyword argument 'fill_value'
```

@Leguark said he doesn't remember even using `fill_value`, so it might be a bug inside of `pandas`. Anyway, I'll start digging into it.

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,7 +35,7 @@ install:
   # Installs from conda-forge
   - conda install -c conda-forge python-graphviz emg3d==0.11
     discretize==0.4.13 pip matplotlib ipython six pytest pytest-cov
-    pytest-mock gdal pandas seaborn>=0.9 qgrid sphinx-gallery
+    pytest-mock gdal pandas==1.0.5 seaborn>=0.9 qgrid sphinx-gallery
     ipywidgets pyevtk arviz dataclasses scikit-image>=0.17
     recommonmark networkx panel setuptools mkl-service
 

